development/build/utils.js

```diff
@@ -70,10 +70,42 @@ function getBrowserVersionMap(platforms, version) {
     const versionParts = [major, minor, patch];
     const browserSpecificVersion = {};
     if (prerelease) {
-      if (platform === 'firefox') {
-        versionParts[2] = `${versionParts[2]}${buildType}${buildVersion}`;
```

Comment:
Firefox's `1.2.3beta1` is now `1.2.3.11001`. Will this cause issues since `version` is probably used to determine if a version is newer than another version? 🤔

Reply:
MMI is only published on Chrome I believe, and Beta was never published to Firefox. So this would only be a concern with Flask.

Details here on how this comparison works for the legacy version format we're using today: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/version/format#comparing_version_parts

This should be fine as long as we release this version change in a minor release. I can't quite tell if this is safe to release as a patch or not. But we don't need to consider that, we can easily avoid releasing this in a patch.

Reply:
cc @danjm @FrederikBolding , just making sure you're aware of this.

I know we wouldn't expect this change to be cherry-picked into a patch release, but if that does happen for some reason, it might break. We should take care not to include this in any patch releases.